commands/pull_request.go

```diff
@@ -423,3 +428,16 @@ func parsePullRequestIssueNumber(url string) string {
 
 	return ""
 }
+
+func fetchMilestoneNumber(client *github.Client, project *github.Project, name string) (int, error) {
```

Comment:
To simplify this method's signature, instead of passing in `client` and `project` to this method, how about passing an already fetched list of milestones?